src/redecomp/transfer/MatrixTransfer.cpp

```diff
@@ -419,7 +439,7 @@ MPIArray<int> MatrixTransfer::buildRecvTrialElemDofs( const RedecompMesh& trial_
     send_trial_elem_dofs[test_r].reserve( send_trial_dof_sizes[test_r] );
     auto n_trial_elems = send_trial_elem_rank[test_r].size();
     for ( int e{ 0 }; e < n_trial_elems; ++e ) {
-      auto trial_elem_dofs = axom::ArrayView<int>(
+      auto trial_elem_dofs = axom::ArrayView<HYPRE_BigInt>(
```

Comment:
Commenting randomly here, but it looks like a simple find and replace swapping an `int` for a `HYPRE_BigInt` wouldn't really work? Seems easy to miss swapping one for the other? Have you combed through this carefully?